src/main/java/io/gingersnapproject/health/CacheBackendReadiness.java

```diff
@@ -4,32 +4,37 @@
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
-
-import io.gingersnapproject.cdc.event.Events;
+import javax.inject.Inject;
 
 import org.eclipse.microprofile.health.Readiness;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.gingersnapproject.cdc.configuration.Configuration;
+import io.gingersnapproject.cdc.event.Events;
+
 @Readiness
 @ApplicationScoped
 public class CacheBackendReadiness extends AbstractHealthChecker<URI> {
    private static final Logger log = LoggerFactory.getLogger(CacheBackendReadiness.class);
    private static final String BACKEND_HEALTH = "Backend Health";
 
+   @Inject
+   Configuration config;
+
    void backendStarted(@Observes Events.BackendStartedEvent ev) {
```

Comment:
It looks like this eventing can be redone as well now as we don't have multiple uris.

Reply:
We don't have multiple URIs, but passing the global one means that the health is still represented as:

```json
        {
            "name": "Backend Health",
            "status": "DOWN",
            "data": {
                "hotrod://127.0.0.1:11223": false
            }
        }
```